src/transport/raw/PeerAddress.h

```diff
@@ -54,19 +54,25 @@ enum class Type : uint8_t
     kBle,
     kTcp,
     kWiFiPAF,
-    kLast = kWiFiPAF, // This is not an actual transport type, it just refers to the last transport type
+    kNfc,
+    kLast = kNfc, // This is not an actual transport type, it just refers to the last transport type
 };
 
+static constexpr uint16_t kUndefinedNFCShortId = 0;
```

Comment:
This seems like an implementation detail, no?  Why is this public?  Should be a private member of PeerAddress.